src/hearings/resolvers/priority/priority.resolve.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve, Router, RouterStateSnapshot } from '@angular/router';
+import { RefDataModel } from 'api/hearings/models/refData.model';
```

Comment:
copy api/hearings/models/refData.model into src/hearings/models/refData.model, node models and ng models need to be separated.